consensus/types/src/beacon_state.rs

```diff
@@ -1477,6 +1614,7 @@ impl<T: EthSpec> BeaconState<T> {
         self.build_all_committee_caches(spec)?;
         self.update_pubkey_cache()?;
         self.build_exit_cache(spec)?;
+        self.build_slashings_cache()?;
 
```

Comment:
just making sure we've intentionally omitted building the `epoch_cache` here?

Reply:
yeah unfortunately because the `epoch_cache` requires `state_processing` stuff to build, we can't build it here in the `types` crate

This is why the `AllCaches` trait exists